cpython-unix/build-cpython.sh

```diff
@@ -494,10 +498,16 @@ make -j ${NUM_CPUS}
 make -j ${NUM_CPUS} sharedinstall DESTDIR=${ROOT}/out/python
 make -j ${NUM_CPUS} install DESTDIR=${ROOT}/out/python
 
-if [ -n "${CPYTHON_DEBUG}" ]; then
-    PYTHON_BINARY_SUFFIX=d
+
+if [ -n "${CPYTHON_FREETHREADED}" ]; then
+    PYTHON_BINARY_SUFFIX=t
+    PYTHON_LIB_SUFFIX=t
 else
     PYTHON_BINARY_SUFFIX=
+    PYTHON_LIB_SUFFIX=
+fi
+if [ -n "${CPYTHON_DEBUG}" ]; then
+    PYTHON_BINARY_SUFFIX="${PYTHON_BINARY_SUFFIX}d"
 fi
```

Comment:
I think we should do flag in lexicographic order, at least i remember them being in this order, even if the PEP doesn't talk about order. I tried to to look in cpython but didn't find anything about order either.

Reply:
huh abiflags say `td` too

Reply:
This ordering matches the order that is produced by CPython; we're not choosing the order, we're just constructing these variables to pull artifacts from the builds.